subsys/bluetooth/host/settings.c

```diff
@@ -281,7 +281,7 @@ static int commit_settings(void)
 	return 0;
 }
 
-SETTINGS_STATIC_HANDLER_DEFINE(bt, "bt", NULL, set_setting, commit_settings, NULL);
+SETTINGS_STATIC_HANDLER_DEFINE_WITH_CPRIO(bt, "bt", NULL, set_setting, commit_settings, NULL, 0);
```

Comment:
IMHO we need to define priorities in a common header file (e.g. somewhere in `subsys/bluetooth/common`) and then use these macros instead of hardcoded values here, in `BT_SETTINGS_DEFINE` and in `mesh/settings.c` and in `BT_MESH_SETTINGS_DEFINE` (if I'm not mistaken).

Reply:
I don't see an appropriate header to add this in, (in `subsys/bluetooth/common`) will therefore create a new one, and I plan to use an enum for the priority macros, makes the most sense to me. Will likely update this PR only late in the day, so any input before to this is welcome :)